### PR TITLE
bump policy-controller to v0.11.0

### PR DESCRIPTION
**Pull request by hectorj2f**
## Description of the change

Bump policy-controller to use v0.11.0

### CODE REVIEW

**Comment by cpanato** [comment]
@hectorj2f need to update the readme

**Review by cpanato — CHANGES_REQUESTED** [review]
readme update

**Review by cpanato — CHANGES_REQUESTED** [review]
version is not quite right

File: charts/policy-controller/Chart.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ sources:
 type: application
 
 name: policy-controller
-version: 0.7.2
-appVersion: 0.8.2
+version: 0.11.0
```
[the remark this answers is not in the document]
Comment (hectorj2f) [reply charts/policy-controller/Chart.yaml:11]: Generally I was thinking to match the app version here considering we skipped couple of versions in between.

[the remark this answers is not in the document]
Comment (cpanato) [reply charts/policy-controller/Chart.yaml:11]: the chart version should not be equal to the app verision, the chart will have other updates that is not related to the app release

but for this case that we are bumping the minor of PC we can bump he minor in the chart version, but not matching the versions

**Comment by avo-sepp** [comment]
Bumping this PR. Would really like to see it completed. 

If the version of 0.11.0 ends up as 0.8 in the Helm chart, do you plan on releasing versions 0.7->0.10 and if so, what Helm chart versions would those get?

**Review by cpanato — APPROVED** [review]

**Pull request merged**